### DISCUSSION TITLE
a post with jupyter notebook | Phu Nguyen Dinh

### DISCUSSION THREAD

**Discussion by giscus** [dinhphu28/blog#10] [Comments]
# a post with jupyter notebook | Phu Nguyen Dinh

an example of a blog post with jupyter notebook

https://blog.dinhphu28.com/blog/2023/jupyter-notebook/

**Comment by dinhphu28** [comment]
Test, This is a comments

**Reply by dinhphu28** [reply]
This is first reply

**Comment by dinhphu28** [comment]
Test 2, this is another comment

**Comment by dinhphu28** [comment]
Test 3, comment after closing discussion
